### ISSUE TITLE
Support for karma 4?

### ISSUE THREAD

**Issue by fr0** [monounity/karma-typescript#317]
I tried using `karma-typescript` `4.0.0-beta.3` and got this warning:

```
warning " > karma-typescript@4.0.0-beta.3" has incorrect peer dependency "karma@1 || 2 || 3".
```

It would be nice if we could get support for karma 4 since it was released 2019-01-23.
